<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>188</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>348</width>
    <height>188</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>348</width>
    <height>188</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>StopWatch</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>139</y>
      <width>184</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>22</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#startButton {
	background-color: rgb(0, 42, 113);
	color: rgb(239, 239, 239);
    border-style: outset;
    border-width: 0px;
	border-color: rgb(239, 239, 239);
    border-radius: 10px;
    min-width: 10em;
    padding: 6px;
}
QPushButton#startButton:pressed {
     background-color: rgb(224, 0, 39);  
 }
 QPushButton#startButton:hover {
     background-color: rgb(151, 212, 0);
	color: rgb(36, 36, 36);   
 }</string>
    </property>
    <property name="text">
     <string>START</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>176</x>
      <y>139</y>
      <width>180</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#resetButton {
	background-color: rgb(0, 42, 113);
	color: rgb(239, 239, 239);
    border-style: outset;
    border-width: 0px;
	border-color: rgb(239, 239, 239);
    border-radius: 10px;
    min-width: 10em;
    padding: 6px;
}
QPushButton#resetButton:pressed {
	background-color: rgb(0, 42, 113);
 }
 QPushButton#resetButton:hover {
     background-color: rgb(165, 0, 13);
	color:rgb(239, 239, 239);
 }</string>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>20</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>62</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>:</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="msecLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>52</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>00</string>
    </property>
   </widget>
   <widget class="QLabel" name="secLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>92</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>00</string>
    </property>
   </widget>
   <widget class="QLabel" name="minutesLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>111</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>92</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>20</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>62</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>:</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-3</x>
      <y>180</y>
      <width>361</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(224, 0, 39);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>361</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>132</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <zorder>line_2</zorder>
   <zorder>startButton</zorder>
   <zorder>resetButton</zorder>
   <zorder>label_2</zorder>
   <zorder>msecLabel</zorder>
   <zorder>secLabel</zorder>
   <zorder>minutesLabel</zorder>
   <zorder>label_3</zorder>
   <zorder>line</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>348</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
